packages/trip-details/src/trip-detail.tsx

```diff
@@ -13,17 +13,8 @@ type Props = {
   summary: ReactElement | string;
 };
 
-// TODO: Remove these two helper methods by moving to semantically correct HTML
-/**
- * Copied from https://stackoverflow.com/questions/50940640/how-to-determine-if-jest-is-running-the-code-or-not
- */
-function isRunningJest() {
-  return process.env.JEST_WORKER_ID !== undefined;
-}
-
+// TODO: Remove this helper method by moving to semantically correct HTML
 function uuidv4(): string {
-  if (isRunningJest()) return "mocked-random-id";
```

Comment:
I'm impressed this isn't causing problems... Why are the snapshots not complaining about changing uuids?

Reply:
no idea why the uuids aren't a problem, I didn't really look into what this was for. I just know it's not doing anything today.

Reply:
Alright...